website_snippet_product_category/readme/USAGE.rst

```diff
@@ -0,0 +1,21 @@
+#. Go to website (frontend)
+#. Press 'Edit' button
+#. Drag the 'Product Category' snippet to the place of the website that you want
+#. Save changes
+
+
+Theming
+~~~~~~~
+
+* `.categ_container` > The container per master category
+* `.categ_tree_level` > The category row (image + text)
+* `.main_tree_level` > The first category row level displayed
+* `.categ_link` > The link of the category row
+* `.categ_img` > The container of the icon of the category row
```

Comment:
Remember this is RST, not MD. Single-backtick is a link; double is code:

```suggestion
* `.categ_container` > The container per master category
* ``.categ_tree_level`` > The category row (image + text)
* ``.main_tree_level`` > The first category row level displayed
* ``.categ_link`` > The link of the category row
* ``.categ_img`` > The container of the icon of the category row
```

BTW thanks for documenting this! :raised_hands:

Reply:
:sob: